src/main/java/bio/terra/service/resourcemanagement/ResourceService.java

```diff
@@ -509,6 +509,10 @@ public void revokePoliciesBqJobUser(String dataProject, Collection<String> polic
     return projectService.markUnusedProjectsForDelete(profileId);
   }
 
+  public void markProjectsForDelete(List<UUID> projectIds) {
+    projectService.markProjectsForDelete(projectIds);
+  }
+
```

Comment:
Can we remove this and the other markProjectsForDelete() methods since we've removed the step?

Reply:
(well, revert it back to the original state, since the billing profile delete still needs the original method)

Reply:
Nevermind - since removing this breaks everything, let's keep it.